### ISSUE TITLE
Snapshots not working

### ISSUE THREAD

**Issue by Antontwo** [reaper-oss/sws#1558]
I tried out the snapshot function on a track as an experiment but it doesn't seem to work. 

I followed Rapid-Fire REAPER Tutorials method but any time I make a change and press "new" for a new snapshot and then go back through the previous mixes (mix 1, mix 2) I get the version I just saved (doing a full track mix).
It's not a complex project. Just me singing on one track. I save it as "new". Mess about with it. Save as "new". But they all end up being the same as the last one I saved as "new" !
And I have the same problem on two computers where "extensions" is installed. It's probably something very simple I have overlooked. A pity because this would be a great help in mixing and so on.

With full track mix ticked all the items underneath are dimmed but it seems to be the same in any video I see where people are explaining the system except for Kenny Goia's groove3 video where the items are not dimmed. 
I asked these questions in the Reaper forum but have so far drawn a blank. Any ideas anyone please?

**Comment by Antontwo** [comment]
Well, I am making some progress.
I have switched to "custom" then ticked everything that is ticked in full track mix, clicked new, changed a lot of things, clicked "new" and now I can switch back to the pre-changes version.

So the fact that the boxes are not ticked in full track mix apparently makes a difference.

Is continuing to work in the custom mode with all the appropriate boxes ticked just the same as full track mix or am I in for a nasty surprise?

**Comment by Antontwo** [comment]
I think my problem was me expecting the snapshot function to do more than it was intended for.

So I was working on one project and trying to work out a vocal harmony, saving each take I took but, silly me,I did not realise it does not work if you add or remove tracks. I was thinking it would update the entire project each time.

In another project, I was fiddling about with Melodyne, expecting these changes but be updated with each mix.


Whereas it simply updates the items listed.
 It was the "full track" that confused me.

Sorry for this silly post
